backend/impl/build.gradle.kts

```
@@ -33,10 +31,10 @@ BuildUtils.vavr(project)
 
 CheckerFrameworkConfigurator.applySubtypingChecker(project)
 
-tasks.withType<JavaExec>() {
-    group = "Execution"
-    description = "Regenerate CLI outputs used for comparison in tests"
-    classpath = sourceSets["test"].runtimeClasspath
-    mainClass.set("com.virtuslab.gitmachete.backend.integration.RegenerateCliOutputs")
-    args = sourceSets["test"].resources.srcDirs.map{ it.absolutePath }.toList()
+tasks.register<JavaExec>("RegenerateCliOutputs") {
```

Uppercase task name? the Gradle convention is to use lowercase:
```suggestion
tasks.register<JavaExec>("regenerateCliOutputs") {
```